code/modules/mob/living/carbon/carbon.dm

```diff
@@ -126,8 +126,7 @@
 		if(isturf(loc))
 			I.remove(src)
 			I.forceMove(get_turf(src))
-			spawn()
-				I.throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),5)
+			I.throw_at(get_edge_target_turf(src,pick(alldirs)),rand(1,3),5)
```

Comment:
`throw_at` uses a system whereby it doesn't need to spawn anymore, at least, as far as I know @tigercat2000